还有残留呀，注意现在的ruff版本为`ruff 0.0.289`
```bash
ruff check .
python/paddle/base/backward.py:763:5: C411 [*] Unnecessary `list` call (remove the outer call to `list()`)
Found 1 error.
[*] 1 potentially fixable with the --fix option.
```

--- a/python/paddle/base/backward.py
+++ b/python/paddle/base/backward.py
@@ -760,7 +760,7 @@ def _remove_no_grad_branch_(
                         ] = grad_op_id_to_fwd_op[op_desc.original_id()]
                     to_insert.append((new_op_desc, idx))
 
-    list([op_descs.insert(p[1], p[0]) for p in reversed(to_insert)])
+    [op_descs.insert(p[1], p[0]) for p in reversed(to_insert)]
 
     return op_descs
 

